controller/src/main/java/io/pravega/controller/store/stream/AbstractStreamMetadataStore.java

```diff
@@ -460,6 +460,12 @@ public OperationContext createContext(String scope, String name) {
         return withCompletion(getStream(scope, stream, context).getActiveTxns(), executor);
     }
 
+    @Override
+    public CompletableFuture<List<ScaleIndicent>> getScaleIncidents(final String scope, final String name,
+                                                                    final OperationContext context, final Executor executor) {
+        return withCompletion(getStream(scope, name, context).getScaleIncidents(), executor);
+    }
```

Comment:
Can you add a Controller service API to call store API? just for consistency, since all the REST APIs primarily call controller service APIs.